src/components/ConversationWindow.js

```diff
@@ -586,6 +593,7 @@ class ConversationWindow extends PureComponent {
                 return <Message 
                     key={message.id} 
                     message={message} 
+                    isMessageByAdmin={ conversationAdmins.includes(message.ownerId) }
```

Comment:
@abhishekbadoni-bigstep we also can make a function `isSentByAdmin` and we can use this function in this component.

Reply:
Done!